### PR TITLE
Update resources from azure-rtos to eclipse-threadx

### PR DESCRIPTION
**Pull request by ericwol-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ Here are the key features and modules of NetX Duo:
...
 
-We also provide [getting started guide](https://github.com/azure-rtos/getting-started) and [samples](https://github.com/azure-rtos/samples) using hero development boards from semiconductors you can build and test with.
+We also provide [getting started guide](https://github.com/eclipse-threadx/getting-started) and [samples](https://github.com/eclipse-threadx/samples) using hero development boards from semiconductors you can build and test with.
 
 See [Overview of Azure RTOS NetX Duo](https://learn.microsoft.com/azure/rtos/netx-duo/overview-netx-duo) for the high-level overview, and all documentation and APIs can be found in: [Azure RTOS NetX Duo documentation](https://learn.microsoft.com/azure/rtos/netx-duo/).
 
-Also there is dedicated [learning path of Azure RTOS NetX Duo](https://learn.microsoft.com/training/paths/azure-rtos-netx-duo/) for learning systematically.
```
Comment (bo-ms) [inline README.md:17]: do you plan to update the link in this PR? azure-rtos-netx-duo

File: README.md [hunk 2]
```diff
@@ -87,15 +87,15 @@ The master branch has the most recent code with all new features and bug fixes.
...
 
 ### Samples
 
 We provide sample codes about how to use various addons in the [`samples`](./samples/) folder.
 
 ### Azure IoT Middleware for Azure RTOS
```
Comment (bo-ms) [inline README.md:96]: Azure IoT Middleware for Eclipse ThreadX

File: README.md (hunk 1 above)
Comment (ericwol-msft) [reply README.md:17]: No, let Tim do that

File: README.md (hunk 2 above)
Comment (ericwol-msft) [reply README.md:96]: Out of scope of this PR

**Review by TiejunMS — APPROVED** [review]

**Review by bo-ms — APPROVED** [review]

**Pull request merged**